bread/__init__.py

```diff
@@ -1 +1 @@
-VERSION = '0.1.1'
+VERSION = '0.1.2'
```

Comment:
Actually I think we should leave the version alone until we do a "release", since we don't know what order we'll end up merging pulls, don't you think?

(And yes I know I bumped the version in a pull earlier. I've changed my mind about that :-) )